docs/development/core/server/kibana-plugin-core-server.deprecationsservicesetup.md

```diff
@@ -31,10 +31,10 @@ async function getDeprecations({ esClient, savedObjectsClient }: GetDeprecations
     // Example of a manual correctiveAction
     deprecations.push({
       title: i18n.translate('xpack.timelion.deprecations.worksheetsTitle', {
-        defaultMessage: 'Found Timelion worksheets.'
+        defaultMessage: 'Timelion worksheets are deprecated'
```

Comment:
Not sure why we didn't catch this in previous PR 🤷

Reply:
Hmm weird indeed